R/data.R

```diff
@@ -55,3 +55,26 @@
 #'    look like the data that one would pull directly an API to access
 #'    patient-level line-list data.
 "syn_nssp_line_list"
+
+#' Incident COVID-19 hospitalisations indexed by the date of positive test
+#'   (reference date) and report date from Germany in 2021 and 2022.
+#'
+#' @format A data.frame with 140,630 rows and 6 columns.
+#' \describe{
+#'   \item{reference_date}{Date of first positive COVID-19 test formatted
+#'    in ISO8601 standards as YYYY-MM-DD.}
+#'   \item{location}{Character string indicating the location of the case
+#'   counts}
+#'   \item{age_group}{Character string indicating the age group of the case
+#'   counts.}
+#'   \item{delay}{Integer specifying the delay, in days, between the reference
```

Comment:
Should we keep this column? With it in place, the user could accidentally not pass a `nowcast_unit` and it would stratify by delay (which throws an error). This might though be helpful to demonstrate?